core/shared/src/main/scala/org/http4s/headers/`Access-Control-Max-Age`.scala

```diff
@@ -18,6 +18,7 @@ package org.http4s
 package headers
 
 import org.http4s.parser.AdditionalRules
+import org.typelevel.ci.CIString
 import org.typelevel.ci.CIStringSyntax
```

Comment:
Just a minor nit: although there's nothing besides CIString and its syntax in the package, then these two imports could be replaced with just a single `import org.typelevel.ci._`

I think it is a suggested way in any case...

Reply:
There is a crowd that believes using wildcard imports makes compilation slower. But I don't mind this change at all. Let's do it.